manim/utils/space_ops.py

```diff
@@ -752,22 +754,45 @@ def earclip_triangulation(verts: np.ndarray, ring_ends: list) -> list:
     return [indices[mi] for mi in meta_indices]
 
 
-def cartesian_to_spherical(vec):
+def cartesian_to_spherical(vec: Sequence[float]) -> np.ndarray:
+    """Returns an array of numbers corresponding to each
+    polar coordinate value (distance, phi, theta).
+
+    Parameters
+    ----------
+    vec
+        A numpy array ``[x, y, z]``.
+    """
     norm = np.linalg.norm(vec)
     if norm == 0:
         return 0, 0, 0
     r = norm
-    theta = np.arccos(vec[2] / r)
-    phi = np.arctan2(vec[1], vec[0])
-    return r, theta, phi
+    phi = np.arccos(vec[2] / r)
+    theta = np.arctan2(vec[1], vec[0])
+    return np.array([r, phi, theta])
+
 
+def spherical_to_cartesian(spherical: Sequence[float]) -> np.ndarray:
+    """Returns a numpy array ``[x, y, z]`` based on the spherical
+    coordinates given.
```

Comment:
Perhaps state the convention of what r, theta and phi mean?

Reply:
How do you mean? I have mentioned their definitions in the parameter section.

Reply:
ah I missed that.